### PR TITLE
net-im/signal-desktop-bin: add new version 1.24.1

### PR DESCRIPTION
**Pull request by robert7k**

### CODE REVIEW

**Comment by juippis** [comment]
No need to answer but just throwing out a comment: Maybe it's time you tried updating EAPI to 7, and replaced gnome2-utils with xdg-utils. With EAPI-7 you can call xdg_icon_cache_update instead of gnome2_icon_cache_update. 

I do not know if the other eclasses support EAPI-7 yet (I have a feeling they do), but you could try it out soonish, or comment why EAPI-7 won't work so it can be brought to attention.

File: net-im/signal-desktop-bin/signal-desktop-bin-1.24.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (mgorny) [inline net-im/signal-desktop-bin/signal-desktop-bin-1.24.1.ebuild]: EAPI 7, please?

File: net-im/signal-desktop-bin/signal-desktop-bin-1.24.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	pax-mark m opt/Signal/signal-desktop
+
+	dosym ../../opt/Signal/${MY_PN} /usr/bin/${MY_PN}
+	dosym ../../usr/lib64/libEGL.so opt/Signal/libEGL.so
+	dosym ../../../usr/lib64/libEGL.so opt/Signal/swiftshader/libEGL.so
+}
+
+pkg_postinst() {
+	xdg_desktop_database_update
+	gnome2_icon_cache_update
```
Comment (mgorny) [inline net-im/signal-desktop-bin/signal-desktop-bin-1.24.1.ebuild]: `xdg_icon_cache_update` is the more modern way. Also you can drop gnome2-utils then.

**Comment by robert7k** [comment]
@mgorny @juippis thank you for the hints, I adjusted accordingly.

**Comment by mgorny** [comment]
Thanks.

**Pull request closed without merging**